libraries/TeamSpeak3/Node/Client.php

```diff
@@ -285,7 +285,7 @@ public function avatarGetName()
    */
   public function avatarDownload()
   {
-    if($this["client_flag_avatar"] == 0) return;
+    if($this["client_flag_avatar"]->toString() == 0) return;
```

Comment:
Rather than returning string, then type casting to int for comparison, I think this should be updated to compare `null` as is done elsewhere in the code:
```php
if($this["client_flag_avatar"] == null) return;
```